.github/startup-test/secrets/datastore_postgres_password

```diff
@@ -0,0 +1 @@
+openslides
```

Comment:
I'd prefer to have these files removed and generate them for the productive startup tests. Having these files present just invites people to use them as their "real" productive secrets, even if they are only in the `.github` folder.